db.go

```diff
@@ -50,6 +48,15 @@ func OpenTestConnection() (db *gorm.DB, err error) {
 			dbDSN = "gorm:gorm@tcp(localhost:9910)/gorm?charset=utf8&parseTime=True&loc=Local"
 		}
 		db, err = gorm.Open(mysql.Open(dbDSN), &gorm.Config{})
```

Comment:
`db` and `DB`

Reply:
I don't understand what you mean

Reply:
https://github.com/go-gorm/sharding/blob/main/examples/order.go

Checkout the difference with this file. You may have a mistake, but I can find it.

Reply:
Sorry,  I can't find my mistake.
function OpenTestConnection return `db `, then assign to global variable `DB`.
```go
func OpenTestConnection() (db *gorm.DB, err error)
```

Reply:
Can I add you on wechat or qq ?